.gitattributes

```diff
@@ -1,5 +1,7 @@
 * text=auto
 /mithril.js binary
 /mithril.min.js binary
+/mithril.mjs binary
```

Comment:
This slip-in change was because #2325 missed it, and I forgot to add it in as part of the review.